metricbeat/metricbeat.full.yml

```diff
@@ -126,6 +126,11 @@ metricbeat.modules:
   kernel.rate_limit: 0
   kernel.include_raw_message: false
   kernel.include_warnings: false
+  kernel.audit_rules: |
+    # Define audit rules here.
+    # Create file watches (-a) or syscall audits (-a or -A). For example:
```

Comment:
Sorry I'm not familiar with audit rules, this comment refers to '-w /etc/passwd` or the next line?